java-checks-test-sources/src/main/java/symbolicexecution/behaviorcache/SpringStringUtilsMethods.java

```diff
@@ -0,0 +1,17 @@
+package symbolicexecution.behaviorcache;
+
+import org.springframework.util.StringUtils;
+
+class SpringStringUtilsMethods {
+
+  String test_isEmpty() {
+    String fileName = null;
+    if (!StringUtils.isEmpty(fileName)) {
```

Comment:
`StringUtils.isEmpty()` seems to be deprecated in latest versions of Spring. They recommend using other methods instead which we should cover as well:
* `org.springframework.util.StringUtils.hasLength(@Nullable String)`
* `org.springframework.util.StringUtils.hasText(@Nullable String)`
* `org.springframework.util.ObjectUtils.isEmpty(@Nullable Object)`

Reply:
All those methods' method behaviors were already implemented. I have just added test cases to cover them.